src/shared/components/layouts/UniProtHeader.tsx

```diff
@@ -127,6 +129,9 @@ const organizationSchema: WithContext<Organization> = {
   '@type': 'Organization',
   url: 'https://www.uniprot.org',
   logo: ['https://www.uniprot.org/android-chrome-512x512.png', Logo],
+  name: 'UniProt',
+  description,
+  sameAs: Object.values(socialUrls),
```

Comment:
```suggestion
  sameAs: Object.values(socialUrls),
  email: 'https://www.uniprot.org/contact',
```
Have you managed to get this bit detected in the Google rich-results tool?
For me it wouldn't appear at all, but when I added an email field it would appear as "Organization", although with a warning about invalid email. I guess it's better than nothing, we could have this and see if it starts appearing in the Google Search Console.
When adding a phone number it would additionally appear as a Local Business, but that's not really what we want to be detected as.

Reply:
Good catch, didn't even notice it organization wasn't being picked up. I can get it to work without a warning using:
```
+  contactPoint: {
+    '@type': 'ContactPoint',
+    url: 'https://www.uniprot.org/contact',
+  },
```

Reply:
Ah, that's way nicer!